common/components/time/template.test.js

```diff
@@ -60,4 +69,80 @@ describe('Time component', function () {
       expect($component.text().trim()).to.equal('Today')
     })
   })
+
+  context('with relative time', function () {
+    let $component
+
+    beforeEach(function () {
+      const $ = renderComponentHtmlToCheerio('time', {
+        datetime: '2019-01-10',
+        relative: true,
+      })
+
+      $component = $('time')
+    })
+
+    it('should contain a datetime attribute', function () {
```

Comment:
This isn't dependent on the `relative` property, so doesn't need to be placed in this `context` block.

Or is this an extra check just to make sure things stay together properly?

Reply:
Yeah, I think I just kept it in there to ensure this attribute is also still there in this instance.